### PR TITLE
[WIP] Add main page

### PR DESCRIPTION
**Pull request by aleksashyn**

### CODE REVIEW

**Comment by kokorin** [comment]
Converted to DRAFT PR

File: client/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@reduxjs/toolkit": "^1.6.0",
     "axios": "^0.21.1",
     "bootstrap": "^5.0.2",
+    "jwt-decode": "^3.1.2",
```
Comment (kokorin) [inline client/package.json:12]: Consider extending & using `/api/v1/users/current` endpoint to get all information about current user.  JWT may contain not all information required and there is no sence in storing avatar URL in JWT.

**Review by kokorin — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 1 above)
Comment (aleksashyn) [reply client/package.json:12]: this is for decode google token

Comment (kokorin) [reply client/package.json:12]: But a user may have dub-manager cookie set, so no login via google would be required.

**Pull request closed without merging**